eng/versioning/version_client.txt

```diff
@@ -5,12 +5,12 @@ com.azure:azure-sdk-all;1.0.0;1.0.0
 com.azure:azure-sdk-parent;1.6.0;1.6.0
 com.azure:azure-client-sdk-parent;1.7.0;1.7.0
 com.azure:azure-ai-textanalytics;1.0.0-beta.1;1.0.0-beta.1
-com.azure:azure-core;1.1.0;1.2.0-beta.1
+com.azure:azure-core;1.1.0;1.2.0
 com.azure:azure-core-amqp;1.0.0-beta.9;1.0.0-beta.9
-com.azure:azure-core-http-netty;1.1.0;1.2.0-beta.1
-com.azure:azure-core-http-okhttp;1.1.0-beta.1;1.1.0-beta.1
+com.azure:azure-core-http-netty;1.1.0;1.2.0
+com.azure:azure-core-http-okhttp;1.1.0-beta.1;1.1.0
```

Comment:
Shouldn't have a dependency on a beta version, also I don't think the beta was ever released so anything referencing this won't be able to find it in Maven. Should change the dependency version to `1.0.0`.

Reply:
https://mvnrepository.com/artifact/com.azure/azure-core-http-netty/1.1.0

Reply:
Was referencing `OkHttp`, I should have clarified that.